### DISCUSSION TITLE
Welcome to ent Discussions!

### DISCUSSION THREAD

**Discussion by a8m** [ent/ent#1333] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by diep-it-dn** [comment]
I would like to share my repo that is using ent with features: Login, social login by google account, register, change password, forgot password (send email to help the user to reset password), JWT, refresh token, posts with tags, integration tests...

https://github.com/diep-it-dn/fullstack-go-angular

I hope it's useful for everyone trying to use ent as a starter/reference project!

**Reply by rotemtam** [reply]
Thanks a lot for sharing, @diep-it-dn!

**Reply by rotemtam** [reply]
I think it's a great example, we would like to share it with the community, if you would be open to that please ping me (rotemtam) on our [slack](https://entgo.io/docs/slack).

**Reply by diep-it-dn** [reply]
Thanks for your feedback! I'm happy to hear that.
I would contribute some things useful to the community, so feel free to share it 👍
